src/NServiceBus.AcceptanceTests/Audit/When_auditing.cs

```diff
@@ -45,8 +46,11 @@ public EndpointWithAuditOff()
                 // Although the AuditTo seems strange here, this test tries to fake the scenario where
```

Comment:
This comment should be adjusted to mention the new name. Maybe also do a quick search to see if there are any others?